.envrc

```diff
@@ -66,13 +66,10 @@ export OKTA_TEST_PASSWORD=
 export OKTA_TEST_SECRET=
 
 # Launch Darkly configuration
-export LD_SDK_KEY=fakekey
-export LD_TIMEOUT_SECONDS=30
-export LD_ENV_USER=6dec0457-d4f0-490b-83ed-4502aa44a95e
-export REACT_APP_LD_ENV_USER=$LD_ENV_USER
-export REACT_APP_LD_USER_HASH=
 export FLAG_SOURCE=LOCAL # LAUNCH_DARKLY, LOCAL, or FILE
-export REACT_APP_LD_CLIENT_ID=5f51549c031fe609e6df43d8
+export LD_SDK_KEY=GO_CHECK_1PASSWORD
```

Comment:
Test request - make this English text

Reply:
Updated in https://github.com/CMSgov/easi-app/pull/1776/commits/c4357105a949ec6f5f7c02493af938dda87af93b.